test/js-native-api/test_general/test_general.c

```diff
@@ -215,7 +215,8 @@ static napi_value testAdjustExternalMemory(napi_env env, napi_callback_info info
   napi_value result;
   int64_t adjustedValue;
 
-  NODE_API_CALL(env, napi_adjust_external_memory(env, 1, &adjustedValue));
+  // Some JavaScript engines may ignore small increments passed to napi_adjust_external_memory
```

Comment:
Can you be more specific? What is the constraint you're seeing?

Reply:
IMO, the node-api adapter for WebKit should normalize small values if this is a WebKit constraint.